test/integration/pruning_test.exs

```diff
@@ -63,6 +64,43 @@ defmodule Oban.Integration.PruningTest do
     :ok = stop_supervised(Oban)
   end
 
+  test "pruning jobs publishes a metric" do
```

Comment:
We don't need separate tests for the telemetry events. The tests effectively duplicate the other pruning tests, and I'd rather avoid all the setup. You can drop these entirely, IMO.